drivers/clock_control/clock_stm32_ll_n6.c

```diff
@@ -293,7 +293,6 @@ static int stm32_clock_control_get_subsys_rate(const struct device *dev,
 		*rate = get_bus_clock(ahb_clock, STM32_APB2_PRESCALER);
 		break;
 	case STM32_CLOCK_BUS_APB4:
-	case STM32_CLOCK_BUS_APB4_2:
```

Comment:
STM32_CLOCK_BUS_APB4_2 corresponds to the RCC_APB4H register (STM32_CLOCK_BUS_APB4 corresponds to RCC_APB4L). Technically speaking STM32_CLOCK_BUS_* designate register addresses. As such, they shouldn't even be used here, and we should only use STM32_SRC_*. But for peripherals that don't explicitly set a source clock, we default to their register clock to calculate their frequency (supposing that their register clock is their default source clock).

So this line should be kept for potential peripherals like that.